bundles/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/internal/handler/AVMFritzBaseThingHandler.java

```diff
@@ -443,7 +517,42 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                     }
                 }
                 if (brightness != null) {
-                    fritzBox.setLevelPercentage(ain, brightness);
+                    if (brightness.equals(BigDecimal.ZERO)) {
+                        fritzBox.setSwitch(ain, false);
+                    } else {
+                        fritzBox.setSwitch(ain, true);
+                        fritzBox.setLevelPercentage(ain, brightness);
+                    }
+                }
+                break;
+            case CHANNEL_COLORTEMPERATURE:
+                BigDecimal color_temperature = null;
```

Comment:
Please respect convention for variable naming. So rather `colorTemperature`.

Reply:
done